python/mxnet/libinfo.py

```diff
@@ -74,6 +131,13 @@ def find_lib_path():
                            'List of candidates:\n' + str('\n'.join(dll_path)))
     if os.name == 'nt':
         os.environ['PATH'] = os.environ['PATH'] + ';' + os.path.dirname(lib_path[0])
+
+    # We're only intersted in the entry at the head of the list...
+    lib_path = lib_path[0:1]
+
+    mxnet_lib_dir = os.path.dirname(lib_path[0])
+    lib_path += _get_ngraph_shared_lib_paths(mxnet_lib_dir)
```

Comment:
This will always fail if mxnet is compiled without nGraph, we need a better solution.